### PR TITLE
feat(stats): refactor to only use once curl and compact code

### PR DESCRIPTION
**Pull request by h3o66**
# Description

Change to remove repeating code from info_stats.sh

## Type of change

* [ ] Bug fix (a change which fixes an issue).
* [ ] New feature (change which adds functionality).
* [ ] New Server (new server added).
* [x] Refactor (restructures existing code).
* [ ] Comment update (typo, spelling, explanation, examples, etc).

## Checklist

PR will not be merged until all steps are complete.

* [ ] This pull request links to an issue.
* [x] This pull request uses the `develop` branch as its base.
* [x] This pull request Subject follows the Conventional Commits standard.
* [x] This code follows the style guidelines of this project.
* [x] I have performed a self-review of my code.
* [x] I have checked that this code is commented where required.
* [ ] I have provided a detailed with enough description of this PR.
* [x] I have checked If documentation needs updating.

## Documentation

If documentation does need updating either update it by creating a PR (preferred) or request a documentation update.
* User docs: https://github.com/GameServerManagers/LinuxGSM-Docs
* Dev docs: https://github.com/GameServerManagers/LinuxGSM-Dev-Docs

**Thank you for your Pull Request!**

### CODE REVIEW

**Review by dgibbs64 — COMMENTED** [review]
Looks good. I will hold until next release as I will need to spend a bit of time to test and ensure working before I merge

**Review by dgibbs64 — APPROVED** [review]

**Comment by dgibbs64** [comment]
I am testing this it take time for the uuid to show. I will merge once confirmed my test server is showing in GA

**Comment by h3o66** [comment]
> I am testing this it take time for the uuid to show. I will merge once confirmed my test server is showing in GA

rebased the branch to the latest develop

**Pull request closed without merging**
